ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/ProfileClustersTable.tsx

```diff
@@ -48,8 +49,20 @@ function ProfileClustersTable({
     onClearFilters,
 }: ProfileClustersTableProps) {
     const { generatePathWithScanConfig } = useScanConfigRouter();
+    const { searchFilter } = useURLSearch();
     const { page, perPage, setPage, setPerPage } = pagination;
 
+    function isComplianceStatusFiltered() {
+        return Array.isArray(searchFilter['Compliance Check Status']);
+    }
+
+    function shouldDisableIcon(statuses) {
```

Comment:
Interesting that I did not find a type for these. The strings correspond to `statusText` properties?
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/compliance.coverage.utils.tsx#L118-L201

And I am unfamiliar enough with search filter to know where else to take advantage of a type.

Reply:
Yeah this bothers me as well. Our backend uses all caps for status labels (e.g., PASS), but when we implemented filtering, we used standard capitalization (e.g., Pass), leading to the inconsistency. I think ideally we should display the status in normal capitalization in the dropdown selection, but convert it to uppercase in the URL parameter..?